src/Microsoft.TemplateEngine.Cli/CommandParsing/FilterOptions.cs

```diff
@@ -0,0 +1,92 @@
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Cli.TemplateResolution;
+using Microsoft.TemplateEngine.Edge.Template;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.TemplateEngine.Cli.CommandParsing
+{
+    internal class FilterOption
```

Comment:
consider sealing the class if you are not designing for inheritance

Reply:
will be inherited in next PR